src/style.css

```diff
@@ -1 +1,53 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 5px;
+}
+
+.nav-list {
+  display: flex;
+  width: 85%;
+  align-items: center;
+  padding: 0 50px 0 0;
+}
+
+.nav__link {
+  box-sizing: border-box;
+  font-family: "Roboto" , sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav-list a:hover {
+  color: #00acdc;
+  transition: 0.3s;
+}
```

Comment:
1 - do not use tags for styles
2 - no need to increase selector specifity, link class is enough
3 - set transition to link class, now it's not working when you remove cursor from link